docker-compose.yml

```diff
@@ -9,13 +9,20 @@ services:
     environment:
       - DISPLAY=:99
       - MESA_GL_VERSION_OVERRIDE=3.3
+      - CLOUD_GAME_WEBRTC_ICEPORTS_MIN=50000
```

Comment:
Is there any risk if the range is this limited? Don't know if there is any recommended range

Reply:
No idea. It can be easily reconfigured changed.